assets/thanos-querier/deployment.yaml

```diff
@@ -15,6 +15,20 @@ spec:
       labels:
         app.kubernetes.io/name: thanos-querier
     spec:
+      affinity:
+        podAntiAffinity:
+          preferredDuringSchedulingIgnoredDuringExecution:
+          - podAffinityTerm:
+              labelSelector:
+                matchExpressions:
+                - key: app.kubernetes.io/name
+                  operator: In
+                  values:
+                  - thanos-querier
+              namespaces:
+              - monitoring
```

Comment:
This is the problem, we have to exclude this change.

Reply:
We have issue to properly configure this in the next release.

Reply:
Should we use previous `kube-thanos` release or should we cut a release branch before https://github.com/thanos-io/kube-thanos/pull/84 and use it here?

/cc @s-urbaniak

Reply:
hmm, didn't we want to mask this out in jsonnet in CMO?

Reply:
i will mask this specific change out here in this PR, i was under the impression we wanted to do this initially @LiliC :thinking:

Reply:
ok, masked this out in https://github.com/openshift/cluster-monitoring-operator/pull/624/commits/e6ba2bff332681ccdc261e71094be4b532fe8992

Reply:
no, because in the previous PR I didn't need that dep, it did not come from kube-prometheus that I was bumping.